ui/app/components/ui/mascot.js

```diff
@@ -43,9 +45,7 @@ export default class Mascot extends Component {
   }
 
   componentDidMount () {
-    const targetDivId = 'metamask-mascot-container'
```

Comment:
Should we remove this id from the HTML too?

Reply:
Sure - I suppose it's not useful anymore.